### PR TITLE
[rlottie] Added setDrawRegion api to Surface for supporting subsurface rendering

### PR DESCRIPTION
**Pull request by smohantty**
#6

### CODE REVIEW

**Comment by hermet** [comment]
hmm How about using such as "drawRegion" / clipRegion / drawRegion instead of viewport?

**Comment by smohantty** [comment]
Have changed the api to setDrawRegion

File: inc/rlottie.h [hunk 1]
```diff
@@ -67,6 +67,24 @@ class LOT_EXPORT Surface {
      */
     Surface(uint32_t *buffer, size_t width, size_t height, size_t bytesPerLine);
 
+    /**
+     *  @brief Sets the Draw Area available on the Surface.
+     *
+     *  Lottie will use the draw region size to generate frame image
+     *  and will update only the draw rgion of the surface.
```
Comment (hermet) [inline inc/rlottie.h]: typo: rgion -> region.

File: inc/rlottie.h [hunk 2]
```diff
@@ -67,6 +67,24 @@ class LOT_EXPORT Surface {
...
+     *
+     *  @param[in] x rigion x position.
+     *  @param[in] y  rigion y position.
```
Comment (hermet) [inline inc/rlottie.h]: x rigion => x region
y  rigion => y region

File: inc/rlottie.h [hunk 3]
```diff
@@ -67,6 +67,24 @@ class LOT_EXPORT Surface {
...
+     *  @param[in] width  rigion width.
```
Comment (hermet) [inline inc/rlottie.h]: please be careful spaces and "region"

File: inc/rlottie.h [hunk 4]
```diff
@@ -67,6 +67,24 @@ class LOT_EXPORT Surface {
...
+     *  @param[in] height  rigion height.
+     *
+     *  @note Default surface format is ARGB32_Premultiplied.
+     *  @note Default draw region  area is [ 0 , 0, surface width , surface height]
```
Comment (hermet) [inline inc/rlottie.h]: region - area

**Pull request merged**